app/components/record_actions/view.rb

```diff
@@ -3,19 +3,21 @@
 module RecordActions
   class View < GovukComponent::Base
     include ApplicationHelper
+    include SummaryHelper
 
     attr_reader :trainee
 
-    def initialize(trainee)
+    def initialize(trainee, missing_fields: false)
       @trainee = trainee
+      @missing_fields = missing_fields
```

Comment:
Worth renaming this to `has_missing_fields` if it's just a boolean?

Reply:
i like your style 👍  will do